Neos.ContentRepositoryRegistry/Classes/ContentRepositoryRegistry.php

```diff
@@ -81,6 +81,14 @@ public function get(ContentRepositoryId $contentRepositoryId): ContentRepository
         return $this->getFactory($contentRepositoryId)->getOrBuild();
     }
 
+    /**
+     * @return array<ContentRepositoryId>
+     */
+    public function getContentRepositoryIds(): array
```

Comment:
thanks i also need this for listing all content repositories via cli or for example for the new 9.0 neos setup :)